references/cuegen/util_test.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2023 The KubeVela Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cuegen
+
+import (
+	gotypes "go/types"
+	"math"
+	"strconv"
+	"testing"
+
+	cueast "cuelang.org/go/cue/ast"
+	cuetoken "cuelang.org/go/cue/token"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIdent(t *testing.T) {
+	tests := []struct {
+		name  string
+		isDef bool
+		want  string
+	}{
+		{name: "test", isDef: true, want: "_#test"}, // invisible (first letter is lowercase)
```

Comment:
Why convert the field with lowercase in the first letter as an invisible def?

Reply:
`ident` comes from the official cue implementation, which believes that def starting with lowercase should only be visible within the CUE package.

Reply:
After some thought, I decided to remove this code. It's useless for us.

Reply:
Removed